src/main/java/org/mamute/dao/QuestionDAO.java

```diff
@@ -278,5 +277,126 @@ public Question questionWith(Attachment attachment) {
 				.setParameter("a", attachment)
 				.uniqueResult();
 	}
-}
 
+	public void delete(Question question) {
```

Comment:
For reference, a logical deletion is fine for us. It's only important that after deletion, things are not visible anymore.